pkg/installer/check.go

```diff
@@ -227,9 +227,11 @@ func (o *Installer) ProviderURLParse(path string, acceptPreRelease bool) (packag
 
 					var buf bytes.Buffer
 					if err = tmp.Execute(&buf, hdPkg); err == nil {
-						packageURL = fmt.Sprintf("https://github.com/%s/%s/releases/download/%s/%s.%s",
-							o.Org, o.Repo, version, buf.String(), getPackagingFormat(o))
-
+						packageURL = fmt.Sprintf("https://github.com/%s/%s/releases/download/%s/%s",
+							o.Org, o.Repo, version, buf.String())
+						if !strings.HasSuffix(packageURL, "tar.gz") || !strings.HasSuffix(packageURL, "zip") {
```

Comment:
I guess the condition should be `and` instead of `or`

Reply:
Made the required changes. Thanks.